pynessie/model.py

```diff
@@ -32,6 +32,11 @@
 
 DETACHED_REFERENCE_NAME = "DETACHED"
 
+CONTENT_ICEBERG_TABLE_NAME = "ICEBERG_TABLE"
```

Comment:
nit: Maybe `ICEBERG_TABLE_TYPE_NAME`? WDYT?

Reply:
on it...